editions/tw5.com/tiddlers/howtos/Custom Styles by title.tid

```diff
@@ -13,15 +13,15 @@ The ~TiddlyWiki core adds several `attributes` to the rendered content. With tho
 For example this tiddler is named: "{{!!title}}" so the attribute looks like this: 
 
 ```
-data-tiddler-title="Custom styles by title"
+data-tiddler-title="Custom styles by data-tiddler-title"
```

Comment:
It seems somebody changed the tiddler names. They are WRONG NOW at tiddlywiki.com. The one in the PR is the right tiddler title! @Jermolene

Reply:
https://tiddlywiki.com/#Custom%20styles%20by%20data-tags is the second tiddler, where the tiddler title seems to be  wrong

Reply:
Thanks @Kartiku @pmario